deps/neverbleed/neverbleed.c

```diff
@@ -1346,20 +1346,51 @@ static void *daemon_conn_thread(void *_sock_fd)
     return NULL;
 }
 
+
+static void neverbleed_cleanup_fds(int listen_fd, int close_notify_fd)
+{
+#ifdef LIBC_HAS_CLOSEFROM
```

Comment:
I appreciate having detection of the libc function and deciding based on that, though the restriction here is that this is a change to neverbleed that does not even have a CMakeFile (or an autoconf file).

Would it be possible to
* check the operating system (e.g., `#if defined(__FreeBSD__) || defined(__NetBSD__) || defined(__OpenBSD__)`) instead, and
* redirect this PR to https://github.com/h2o/neverbleed?

FWIW, we do that occasionally (as an example, see https://github.com/h2o/h2o/blob/master/lib/common/socket/evloop.c.h#L32-L86), and I do not mind having another.

Reply:
Completed. Moved to:

https://github.com/h2o/neverbleed/pull/29